src/apps/logging/logging.cpp

```diff
@@ -48,6 +48,44 @@ namespace fmt
       return format_to(ctx.begin(), "{}", fmt::join(vr, "\n\t"));
     }
   };
+
+  template <>
```

Comment:
Minor: I find that the formatting code makes our sample application very verbose and difficult for users to understand how our apps work. Could we move this to a different file to make our sample app as straightforward as possible?

Reply:
Sure - I've moved these into `apps/logging/formatters.h` so the sample app is more readable.